### DISCUSSION TITLE
Project player addons

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#4197] [Discussion]
Should project player be moved to a top-level category as suggested in #3014 or stay a subcategory of editor addons? I think it should be a new category, but there some project player addons that only work in the editor like the clone counter and others that others that affect the player, but also other parts of the editor like drag and drop.

Related discussion: #3014, #3955, #3899, #3912

**Comment by mxmou** [comment]
I think addons that are part of the project player but only work in the editor should stay in the editor features category. That's where someone who doesn't know that their UI is placed next to the stage would probably look for them.

**Reply by DJ-Laser** [reply]
It should be in a subcatagory maybe

**Reply by WorldLanguages** [reply]
"Project player" is already a subcategory

**Comment by WorldLanguages** [comment]
I don't think it should be moved, at least for now

**Reply by Samq64** [reply]
Have you changed your mind about this? You added #3955 to the 1.33 milestone.

**Reply by WorldLanguages** [reply]
@Samq64 I'm more or less indifferent and I believe @mxmou is completely in favor of creating this new category, so I think it could be merged for 1.33
